alot/commands/__init__.py

```diff
@@ -22,6 +22,20 @@ def __init__(self):
         self.undoable = False
         self.help = self.__doc__
 
+        mode, cmdname = reverse_lookup_command(self.__class__)
```

Comment:
... ah, or do you mean store them locally to make the code cleaner? I'll do that.